app/assets/javascripts/lib/pulltorefresh.js

```diff
@@ -54,319 +73,352 @@
 
   try {
     window.addEventListener('test', null, {
-      get passive() { // eslint-disable-line getter-return
-        _shared.supportsPassive = true;
+      get passive() {
+        // eslint-disable-line getter-return
```

Comment:
this should be on the line of the `get passive`, or you can put it in the line before that and add `eslint-disable-next-line getter-return`

Reply:
It was on the line of the function, however, lint-staged somehow changes this to be the line below it. I moved it to be above the line now. However, the error still shows.